### ISSUE TITLE
Release v

### ISSUE THREAD

**Issue by mkllnk** [openfoodfoundation/openfoodnetwork#10731]
## Preparation on Thursday

- [x] Merge pull requests in the [Ready To Go] column
- [ ] Include translations: `tx pull --force`
- [ ] [Draft new release]. Look at previous [releases] for inspiration.
- [ ] Notify [#instance-managers] of user-facing changes.

## Testing

- [ ] [Find build] of the release commit and copy it below.
- [ ] Move this issue to Test Ready.
- [ ] Notify `@testers` in [#testing].
- [ ] Test build: 

## Finish on Tuesday

- [ ] Publish and notify [#global-community] (this is automatically posted with a plugin)
- [ ] Deploy the new release to all managed instances.
  <details><summary>Command line instructions</summary>
  <pre>
  cd ofn-install
  git pull
  ansible-playbook --limit all-prod --extra-vars "git_version=vx.y.z" playbooks/deploy.yml
  </pre>
  </details>
- [ ] Notify [#instance-managers]:
  > @instance_managers The new release has been deployed.
- [ ] Nudge next release manager

The full process is described at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Releasing.

[Ready To Go]: #zenhub
[Transifex pull request]: https://github.com/openfoodfoundation/openfoodnetwork/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+head%3Atransifex
[Draft new release]: https://github.com/openfoodfoundation/openfoodnetwork/releases/new?tag=v&title=v+Code+Name&body=Congrats%0A%0ADescription%0A%0A%23%23+User+facing+changes+:eyes:%0A%0A%0A%0A%23%23+Technical+changes+:wrench:%0A%0A
[releases]: https://github.com/openfoodfoundation/openfoodnetwork/releases
[#instance-managers]: https://app.slack.com/client/T02G54U79/CG7NJ966B
[#testing]: https://openfoodnetwork.slack.com/app_redirect?channel=C02TZ6X00
[Find build]: https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/master
[#global-community]: https://app.slack.com/client/T02G54U79/C59ADD8F2

**Comment by sigmundpetersen** [comment]
Shouldn't we bump the minor version number on this if this is releasing Split Checkout for all?

**Comment by RachL** [comment]
This is ready to go :) https://docs.google.com/document/d/1CRnS_6vpUfPintK1W9oxSHSGHIsU2THbww17ALgGLps/edit#

For the version number I have no strong opinion. We can also bump the number when we remove completely the old code.

**Comment by mkllnk** [comment]
> Shouldn't we bump the minor version number on this if this is releasing Split Checkout for all?

Good point, I forgot about that. The rules of bumping version numbers are not very clear for OFN though. This is what we've done in the past:

- v4.3.0 This release introduce a new version of Rails (7.0)!
- v4.2.0 breaking change around image storing.
- v4.1.0 ???
- v4.0.0 We're switching to Puma
- v3.8.0 Rails 6.0
- v3.7.0 Rails 5.2
- v3.6.1 Rails 5.0
- v3.5.0 this release requires various bits of configuration from ofn-install so provisioning beforehand is mandatory.
- v3.4.0 ???
- v3.3.0 need to regenerate some of their images, as we are now using a new size.
- ...
- v3.0.0 major upgrades for two of our core dependencies, bringing us to Spree 2.1 and Rails 4.0
- v2.0.0 upgrade from Spree 1 to Spree 2

It looks like we have been using the version numbers for technical changes which are not backwards compatible like Rails upgrades or require provisioning with ofn-install. The split checkout is completely reversible and doesn't require any additional sysadmin action. Version numbers are not necessarily communicated to users.

I'll keep the patch version number.
